include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -32,7 +32,7 @@
 #endif
 
 // Macros to check the new SYCL features
-#define _ONEDPL_NO_INIT_PRESENT (__LIBSYCL_VERSION >= 50300)
+#define _ONEDPL_NO_INIT_PRESENT (__LIBSYCL_VERSION >= 50200)
```

Comment:
LIBSYCL_VERSION became 50200 before deprecation of noinit and introduction of no_init. You can find commits and dates in the description of #298.